draft-ietf-dnsop-svcb-https.md

```diff
@@ -1899,7 +1906,7 @@ long, it is broken into several lines.
     \x02\x9b                                           # key 667
     \x00\x09                                           # length 9
     hello\xd2qoo                                       # value
-{: title="A generic key and quoted value with a decimal escape"}
+{: title="A generic key and quoted value with an octal escape"}
```

Comment:
I'm not sure what "decimal escape" was intended to mean; this is a best guess

Reply:
Nope, it's actually a decimal!  RFC 1035 Section 5.1:

~~~
\DDD            where each D is a digit is the octet corresponding to
                the decimal number described by DDD.  The resulting
                octet is assumed to be text and is not checked for
                special meaning.
~~~

Zone files actually don't support hex, octal, or unicode escaping.

Reply:
Oops, reverted.
I ... am not really sure why my brain saw 1-255 and somehow got "octal" from that :-/

Reply:
I'd guess "resulting octet" got slurred in "resulting octal value."